### PR TITLE
[REVIEW] opg_degree_update

### PR DESCRIPTION
**Pull request by Iroy30**

### CODE REVIEW

File: cpp/src/structure/graph.cu [hunk 1]
```diff
@@ -36,7 +36,7 @@ void degree_from_offsets(vertex_t number_of_vertices,
 }
 
 template <typename vertex_t, typename edge_t>
-void degree_from_vertex_ids(const cugraph::experimental::Comm &comm,
+void degree_from_vertex_ids(const raft::handle_t *handle,
```
Comment (afender) [inline cpp/src/structure/graph.cu:39]: I believe it is passed by reference in cuML. 
We may want to do that for consistency or does it have to be a pointer?

Comment (Iroy30) [reply cpp/src/structure/graph.cu]: updated

**Comment by Iroy30** [comment]
rerun tests

**Comment by Iroy30** [comment]
rerun tests

**Comment by Iroy30** [comment]
rerun tests

**Comment by afender** [comment]
Just need CI to pass and fix changelog merge conflicts
Python tests complain about` ModuleNotFoundError: No module named 'cugraph.dask.common'`

**Comment by Iroy30** [comment]
rerun tests

**Pull request merged**